### PR TITLE
Remove redundant check and fix language in comment

### PR DESCRIPTION
**Pull request by enzowritescode**
#### Summary
- Remove redundant if statement that was already handled by `if csrfCheckNeeded { ...`
- Change language in comment per IDE recommendation

#### Ticket Link
NA

#### Screenshots
condition-always-true

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: server/channels/web/handlers.go [hunk 1]
```diff
@@ -509,11 +509,6 @@ func (h *Handler) checkCSRFToken(c *Context, r *http.Request, token string, toke
 			sid := ""
 			userId := ""
 
-			if session != nil {
```
Comment (JulienTant) [inline server/channels/web/handlers.go:512]: So I see that that reaching this point implies that `session != nil` so the check was not needed, but we probably want to preserve the assigned values (or use them directly in the log fields declaration bellow)

**Review by JulienTant — COMMENTED** [review]
Looks good overall @enzowritescode, but this change makes us loose some context in the logs.

File: server/channels/web/handlers.go (hunk 1 above)
Comment (enzowritescode) [reply server/channels/web/handlers.go:512]: oh shoot good catch.

**Review by JulienTant — APPROVED** [review]

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

**Pull request merged**